src/api/app/controllers/webui/user_controller.rb

```diff
@@ -54,9 +54,10 @@ def show
     @owned = @displayed_user.owned_packages
     @groups = @displayed_user.groups
     @role_titles = @displayed_user.roles.global.pluck(:title)
+  end
 
-    return unless User.current == @displayed_user
-    @is_displayed_user = User.current == @displayed_user
+  def tasks
```

Comment:
Can we move that to a dedicated controller, e.g. ``Webui::User::TasksController`` and then create a show / index action?

Reply:
@ChrisBr done